src/components/POLine/Vendor/VendorForm.js

```diff
@@ -45,12 +62,18 @@ const VendorForm = ({ order, accounts }) => {
 };
 
 VendorForm.propTypes = {
+  change: PropTypes.func.isRequired,
+  dispatch: PropTypes.func.isRequired,
   accounts: PropTypes.arrayOf(PropTypes.object),
   order: PropTypes.object.isRequired,
+  vendorRefNumberType: PropTypes.string,
+  vendorRefNumber: PropTypes.string,
 };
 
 VendorForm.defaultProps = {
   accounts: [],
+  vendorRefNumber: '',
```

Comment:
I'm not sure if need these values at all, it's handled by redux form and initialValues